lib/SILGen/SILGenPoly.cpp

```diff
@@ -1518,9 +1522,9 @@ class ResultPlanner {
     // information we needed.
     assert(data.OuterResults.empty());
     assert(data.InnerResults.empty());
-    assert(data.InnerIndirectResultAddrs.size()
-           == SILFunctionConventions(innerFnType, Gen.SGM.M)
-                  .getNumIndirectSILResults());
+    assert(data.InnerIndirectResultAddrs.size() ==
```

Comment:
FWIW my git clang-format script does this`--style "{BasedOnStyle: LLVM, BreakBeforeBinaryOperators: NonAssignment}"` because I find asserts *much* easier to read that way. I haven't convinced everyone else to do it though.